class DataExistsError(Exception):
    pass


class ApiAuthenticateError(Exception):
    pass
